src/test/java/features/loginTest.feature

```diff
@@ -1,22 +1,18 @@
 Feature: Application login
 
   Scenario: log in using valid credentials
-    Given User is on the login page - verify "Welcome, Please Sign In!" displayed on the page
-    When user enter valid credentials and clicks on the Login button
+    Given User is on the login page
+    When user enters valid credentials - email "syst@email.com" and password "testtest123", and clicks on the Login button
     Then Home page appears
 
-  Scenario: log in using invalid email and valid pwd
-     Given User is on the login page - verify "Welcome, Please Sign In!" displayed on the page
-     When user enters invalid email and valid pwd and clicks on the Login button
-     Then error message "No customer account found" appears
+    Scenario Outline: log in using invalid credentials
+      Given User is on the login page
+      When user logs in with login <Username> and password <Password>
```

Comment:
You can usev  same step to enter valid and invalid credentials. No need to have two different steps.